core/instance.go

```diff
@@ -225,12 +242,15 @@ func (i *instance) getCheapestCompatibleSpotInstanceType() (string, error) {
 		logger.Println("Comparing ", candidate.instanceType, " with ",
 			current.instanceType)
 
+		currentPrice := i.calculatePrice(candidate)
```

Comment:
I would like to have this named something like candidatePrice, in order to match the naming of the candidate variable used in this block of code.